independent-publisher-connectors/MyHospitals by AIHW/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2817 @@
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "GetMeasureDownloads",
+              "value-path": "Name",
+              "value-collection": "results",
+              "value-title": "Name",
+              "parameters": {
+                "Accept": "application/json"
+              }
+            }
+          }, {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+            "x-ms-visibility": "internal"
+          }
+        ],
+        "summary": "Get Measure Across Reporting Unit Download",
+        "description": "Gets a measure across reporting unit download, in XLSX format, for the supplied measure download code.",
+        "operationId": "GetMeasureReportingunitDownload"
+      }
+    },
+    "/reporting-units-downloads/datasheet-codes": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "result": {},
+                "version_information": {
+                  "$ref": "#/definitions/VersionInformation"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get Reporting Unit Datasheet Codes Available",
+        "description": "Gets dictionary, indexed by the datasheet code, of all available reporting unit datasheet codes and descriptions that can be used in the reporting unit download API call.",
+        "operationId": "GetReportingUnitDatasheetCodes",
+        "parameters": [{
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal",
+            "x-ms-url-encoding": "single"
+          }
+        ]
+      }
+    },
+    "/reporting-units-downloads/mappings": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "Get Reporting Unit Mapping Download",
+        "description": "Gets the reporting unit mapping download, in XLSX format.",
+        "operationId": "GetReportingUnitMappingDownload",
+        "parameters": [{
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/reporting-units-downloads/{datasheet-code}/{reporting-unit-code}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {}

```

Comment:
@paulculmsee Empty response schema is not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide a valid response schema definition for all the operations in the swagger.